src/components/Form/CheckboxRaw.js

```diff
@@ -117,6 +124,7 @@ CheckboxRaw.propTypes = {
    * adds required attribute to the checkbox
    */
   required: PropTypes.bool,
+  wrapperClassName: PropTypes.string,
```

Comment:
Please add doc (even if it might seem clear what this does)

Reply:
added